react/week3/todolist/src/Components/Todo.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import ChildrenWraper from "./ChildrenWraper";
+import PropTypes from 'prop-types';
+import {FaRegCalendarAlt} from 'react-icons/fa'
+import {MdOutlineLabelImportant} from 'react-icons/md'
+import {FiDelete, FiEdit3} from 'react-icons/fi'
+
+const Todo = ({todo, toggleComplete, removeTodo}) => {
+
+  const {description, deadline, id, completed} = todo
+
+  const [edit, setEdit] = useState(true);
+  const [editedDescription, setEditedDescription] = useState(description)
+  const [newDeadline, setNewDeadline] = useState(deadline)
+
+  const handleCheckboxClick = () => {
+    toggleComplete(id);
+  }
+
+  const handleRemoveClick = () => {
+    removeTodo(id);
+}
```

Comment:
As you're not doing anything else with the data I can recommend using the functions nested within directly on the event handlers `onClick={() => toggleComplete()}`

Reply:
Thank you for the feed back I will implement your suggestions asap 👍